app/src/main/java/app/DTO/Player.kt

```diff
@@ -1,9 +1,18 @@
 package app.DTO
 
+
+/**
+ * The player object to store data.
+ * @param Energy the amount of energy the player has.
+ * @param Money the amount of money the player has.
+ * @param Status the status of the player.
+ * @param Day the day the player is currently on.
+ * @constructor creates a prompt with the given parameters.
+ */
 data class Player(
-    var Energy: Int, //How much energy the player has for the day
-    var Money: Int,  //How much money the player has
-    var Status: Int, //The players status
-    var Day: Int     //Which day the player is on
+    var Energy: Int,
+    var Money: Int,
+    var Status: Int,
+    var Day: Int
```

Comment:
Good call.  Better to put the variable descriptions in the KDoc, than inline.  KDoc is much preferred to inline; you can generate documentation from it.